examples/train_model_cluster.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# conda work environment activated
+source /groups/icecube/${USER}/.bashrc
+cd /groups/icecube/${USER}/graphnet_user/env/
+conda activate gnn_py38_leon
```

Comment:
This does presuppose that the required conda environment is available and currently that the path exists.
This is currently an issue I encountered on the NPX cluster as .bashrc does not exist, only bash_history.

I am currently in the process of reiterating this script to be more general. I will try and add a submission checker

Reply:
Just bumping this. I think it would be good to have the script which is executed be as self-contained as possible. We might, e.g., want to create a conda environment on the fly, install the necessary packages, and delete the conda environment or something like that, to make sure that we don't have to pre-install the package on the cluster nodes on which we want to run.